framework/src/base/MaxQpsThread.C

```diff
@@ -37,7 +38,8 @@ MaxQpsThread::MaxQpsThread(MaxQpsThread & x, Threads::split /*split*/) :
     _qtype(x._qtype),
     _order(x._order),
     _face_order(x._face_order),
-    _max(x._max)
+    _max(x._max),
+    _max_shape_funcs(x._max_shape_funcs)
```

Comment:
I thought about it but didn't want to cause too much churn. I'm happy to rename it though, it may save some developer some time in the future.